lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -0,0 +1,28 @@
+module KubernetesDeploy
+  class Cloudsql < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
Yes.

That KubernetesResource was more or less cargo culted from the other examples.